resident/resident-service/src/test/resources/application.properties

```diff
@@ -33,65 +170,120 @@ resident.clientId=mosip-resident-client
 # Kernel auth secret key for IDA
 resident.secretKey=abc123
 #Token genration issue url
+mosip.keycloak.issuerUrl=http://keycloak/auth/realms/mosip
 token.request.issuerUrl=${mosip.keycloak.issuerUrl}
 
-#--------------URI--------------------------
-#Auth Services
-KERNELAUTHMANAGER=http://kernel-auth-service/v1/authmanager/authenticate/clientidsecretkey
-REGPROCPRINT=http://regproc-print-service/registrationprocessor/v1/print/uincard
-INTERNALAUTH=http://ida-internal-service/idauthentication/v1/internal/auth
-INTERNALAUTHTRANSACTIONS=http://ida-internal-service/idauthentication/v1/internal/authTransactions
-KERNELENCRYPTIONSERVICE=http://ida-internal-service/idauthentication/v1/internal/publickey
-IDAUTHCREATEVID=http://idrepo-vid-service/idrepository/v1/vid
-IDAUTHREVOKEVID=http://idrepo-vid-service/idrepository/v1/vid/deactivate
-#Notification service
-IDREPOGETIDBYUIN=http://idrepo-identity-service/idrepository/v1/identity/idvid
-IDREPOGETIDBYRID=http://idrepo-identity-service/idrepository/v1/identity/idvid
-GETUINBYVID=http://idrepo-vid-service/idrepository/v1/idvid
-resident.notification.emails=mosiptestuser@gmail.com
-resident.notification.message=Notification has been sent to the provided contact detail(s)
+
+#------Temporary | remove this --------------------------
```

Comment:
Why does few comment says remove this ? do we need all these properties to run the open api plugin ?

Reply:
No, we don't need these i have removed this in latest commit.

Reply:
ok resolved